docs/docker-integration/cloudformation.rst

```diff
@@ -6,9 +6,12 @@
         .toctree-wrapper { display:none; }
     </style>
 
-================================================================
-Installing Flocker with Docker Swarm on AWS using CloudFormation
-================================================================
+====================================
+Installing Flocker with Docker Swarm 
+====================================
+
+on AWS using CloudFormation
```

Comment:
Just noting that this is not the same split as on the image on the issue.

That split was:
```
Installing Flocker
with Docker Swarm on AWS using CloudFormation
```
Is there intentionally a difference in where you are splitting the line?

Also maybe use re-structured-tesxt's subtitle feature? http://docutils.sourceforge.net/docs/user/rst/quickstart.html#document-title-subtitle

Seems more representative of the intent of this text.